fvm/src/machine/mod.rs

```diff
@@ -20,7 +20,7 @@ pub use default::DefaultMachine;
 
 mod engine;
 
-pub use engine::{Engine, MultiEngine};
+pub use engine::{Engine, EngineConfig, MultiEngine};
```

Comment:
This is used [here](https://github.com/filecoin-project/ref-fvm/blob/927394b0be886e4d4685c511fb1d87a96fe35b2f/fvm/src/lib.rs#L255) to give test zero cost wasm executon.

If this was intended to not be public, I can hide this under a cfg for testing only. I would rather have more flexibility with gas prices and usage in tests than using gas prices loaded from the network version.

Reply:
It makes sense to make this public.